src/backend/executor/nodeAgg.c

```diff
@@ -2092,6 +2095,25 @@ hash_agg_update_metrics(AggState *aggstate, bool from_tape, int npartitions)
 			}
 		}
 	}
+
+	if(DEBUG1 >= log_min_messages && aggstate->streaming)
+	{
+		for (int setno = 0; setno < aggstate->num_hashes; setno++)
+		{
+		AggStatePerHash perhash = &aggstate->perhash[setno];
+		if (perhash->chain_count > 0)
+		{
+			elog(DEBUG1, "hash_agg_update_metrics streaming: setno=%d, chain_length_total=%.1f, chain_length_max=%d, bucket_used=%lu, bucket_total=%lu, num_expansions=%u",
+				 setno, 
+				 (double)perhash->chain_length_total / perhash->chain_count,
+				 perhash->chain_length_max,
+				 perhash->bucket_used,
+					perhash->bucket_total,
+					perhash->num_expansions);
+			}
+		}
+	}
+
```

Comment:
1. codes indent
2.Is it advisable to log messages in the executor? Could this introduce overhead during execution?